### PR TITLE
Houdini: Redshift ROP image format bug

### PR DESCRIPTION
**Pull request by mustafa-zarkash**
Problem : 
"RS_outputFileFormat" parm value was missing 
and there were more "image_format" than redshift rop supports 

Fix: 
1)  removed unnecessary formats from `image_format_enum`
2) add the selected format value to `RS_outputFileFormat`

### CODE REVIEW

**Comment by mustafa-zarkash** [comment]
Task linked: [OP-6269 Houdini Redshift ROP instance image format bug](https://app.clickup.com/t/6658547/OP-6269)

File: openpype/hosts/houdini/plugins/create/create_redshift_rop.py [hunk 1]
```diff
@@ -3,12 +3,12 @@
 import hou  # noqa
 
 from openpype.hosts.houdini.api import plugin
-from openpype.pipeline import CreatedInstance
```
Comment (moonyuet) [inline openpype/hosts/houdini/plugins/create/create_redshift_rop.py:6]: If it is not included, please remove the comment in line 31 `# type: CreatedInstance`. thanks!

**Review by moonyuet — APPROVED** [review]
code looks good and i did a test with redshift rop in houdini. The image format is set as expected. 

image

Review the comment above and update the branch :)

**Review by moonyuet — APPROVED** [review]
LGTM

**Review by mkolar — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**